tilt_controller_3:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: tilt_joint
    joint_speed: 2.0
    motor:
        id: 3
        init: 0
        min: 0
        max: 4095

